library/std/src/sys/process/unix/unix.rs

```diff
@@ -292,6 +287,11 @@ impl Command {
             cvt_r(|| libc::dup2(fd, libc::STDERR_FILENO))?;
         }
 
+        // SAFETY: Dropping `stdio` (ChildPipes) is safe here since ChildPipes
+        // does not involve heap memory allocations
+        debug_assert!(size_of::<ChildPipes>() <= 24);
+        drop(stdio);
```

Comment:
Agreed with joboet that we shouldn't need this. If anything, you could do a debug_assert with F_GETFL that ensures F_CLOEXEC is set.

Reply:
yeah, I changed the PR to update outdated comments and add debug_assertion.